src/commands/Patrons/mostdrops.ts

```diff
@@ -27,7 +27,7 @@ export default class extends BotCommand {
 			}[]
 		>(query);
 
-		if (result.length === 0) throw `No results found.`;
+		if (result.length < 10) throw `Not enough of this item have been dropped yet.`;
```

Comment:
Just anonymize it. If theres less than 10, show `Anonymous` as the username instead.